### PR TITLE
Create CodeReview

### PR DESCRIPTION
**Pull request by cullenbmacdonald**

### CODE REVIEW

File: Eng/CodeReview.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Code Review
+All of our engineering work makes it to production by first passing through a
+code review. No line of code is too small or inconsequential to bypass this part
+of our product lifecycle.
+
+## Opening a Pull Request
```
Comment (KevinMiller-TheWing) [inline Eng/CodeReview.md:6]: I think these are great guidelines! I would suggest consistency in final punctuation   🤓  🤓  🤓

File: Eng/CodeReview.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+When opening a pull, a few things will help get quick and actionable feedback.
+1. Use the PR template that is added by default.
+ - Reference the ticket
+ - provide a test plan with relevant login info and environment
```
Comment (KevinMiller-TheWing) [inline Eng/CodeReview.md:10]:
```suggestion
 - Provide a test plan with relevant login info and environment
```

**Review by jain-ruchi — APPROVED** [review]
looks great! optionally we can add something about how if you're assigned as reviewer, as a rule of thumb, you shouldn't let 24 hours pass before reviewing unless you're out

**Review by nataliecodes — APPROVED** [review]
i like this!

File: Eng/CodeReview.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+2. Add at least two code reviewers.
+ - One engineer within your squad
+ - One engineer from outside your squad
+ - Additional folks if you want extra eyes.
+3. Give yourself a code review
```
Comment (krystalraphael) [inline Eng/CodeReview.md:17]: This is great, can we add a note here to add a status tag to indicate the code is ready i.e. `Status: Code Review`

Also, as part of the Give yourself a code review to test points of potential impact on other flows. If DB work, ensure its compatible across backend applications (Meteor and API). Changes to the API? Will this break support for iOS or Web?

**Review by krystalraphael — APPROVED** [review]

**Pull request merged**